server/util.go

```diff
@@ -354,6 +354,9 @@ const (
 
 func appendFormatFloat(in []byte, fVal float64, prec, bitSize int) []byte {
 	absVal := math.Abs(fVal)
+	if absVal > math.MaxFloat64 || math.IsNaN(absVal) {
```

Comment:
Should we add warnings in this case?

Reply:
@lzmhhh123 Do you mean the warnings return to client? MySQL returns no warning in this case:

```
root@127.0.0.1 : gangshen 12:23:12> select sum(md5int) from t1;
+-------------+
| sum(md5int) |
+-------------+
|           0 |
+-------------+
1 row in set (0.02 sec)
```